examples/amp-user-location.amp.html

```diff
@@ -0,0 +1,232 @@
+        }
+      },
+      "clickPingsPoll": {
+        "on": "click",
+        "selector": "${selector}",
+        "request": "poll",
+        "vars": {
+          "type": "click",
+          "selector": "body"
+        }
+      },
+      "timer": {
+        "on": "timer",
+        "timerSpec": {
+          "interval": 5,
+          "maxTimerLength": 20
+        },
+        "request": "event",
+        "vars": {
+          "type": "timer"
+        }
+      }
+    }
+  }
+  </script>
+</amp-analytics>
+
+<h2>amp-list usage without amp-bind</h2>
+<p>
+  This example populates variable substitution for analytics
+  and triggers the AMP actions and events.
+</p>
+<amp-user-location on="
+  approve: list-form.submit;
+  deny: denied5.show, requesting5.hide;
+  error: errored5.show, requesting5.hide;" id="location5" layout="nodisplay">
+</amp-user-location>
+<fieldset>
+  <legend>List form</legend>
+  <button on="tap: requesting5.show, location5.request();">Use my location</button>
+  <span hidden id="denied5">Denied!</span>
+  <span hidden id="errored5">Errored!</span>
+  <span hidden id="requesting5">Please wait</span>
+  <form
+      id="list-form" method="POST" action-xhr="/form/echo-json/post"
+      on="submit-success: requesting5.hide, AMP.setState({listData: event.response})">
+    <input type="hidden" name="latlon" data-amp-replace="AMP_USER_LOCATION" value="AMP_USER_LOCATION">
+    <input type="hidden" name="lat" data-amp-replace="AMP_USER_LOCATION" value="AMP_USER_LOCATION(LAT)">
+    <input type="hidden" name="lon" data-amp-replace="AMP_USER_LOCATION" value="AMP_USER_LOCATION(LON)">
+    <input hidden type="checkbox" checked name="businesses" value="McCarlos">
+    <input hidden type="checkbox" checked name="businesses" value="JCarlos">
+    <input hidden type="checkbox" checked name="businesses" value="Carlos O'Vializ's">
+  </form>
+  <amp-list [src]="listData" layout="responsive" width="16" height="9" single-item>
+    <template type="amp-mustache">
+    <p>Businesses near <b>{{lat}} {{lon}}</b></p>
+    <ul>
+      {{#businesses}}<li>{{.}}</li>{{/businesses}}
+    </ul>
+    </template>
+  </amp-list>
+</fieldset>
+
+
+<h2>amp-list usage with amp-bind</h2>
+<p>
+  This example populates variable substitution for analytics
+  and triggers the AMP actions and events.
+</p>
+<amp-user-location on="
+  approve: AMP.setState({listLat: event.lat, listLon: event.lon}), requesting6.hide;
+  deny: denied6.show, requesting6.hide;
+  error: errored6.show, requesting6.hide;" id="location6" layout="nodisplay">
+</amp-user-location>
+<button on="tap: requesting6.show, location6.request();">Get my local deals</button>
+<span hidden id="denied6">Denied!</span>
+<span hidden id="errored6">Errored!</span>
+<span hidden id="requesting6">Please wait</span>
+<amp-list
+    src="https://localhost:8000/user-location/general-deals"
+    [src]="'https://localhost:8000/user-location/localized-deals/TODO?lat=' + listLat + '&lon=' + listLon"
```

Comment:
This also may warn per different default bound/unbound `src`. If we're showing this as an example, it might be a good idea to keep zero warnings and restructure dev server routing so the `src`s match.